Tests/Functional/Observation/All/main.swift

```diff
@@ -74,10 +78,10 @@ class Observation: XCTestCase {
         XCTAssertEqual(observer.finishedBundlePaths.count, 0)
 
         XCTFail("fail!")
-        XCTAssertEqual(observer.failureDescriptions, ["failed - fail!"])
+        XCTAssertEqual(observer.failureDescriptions, ["Observation.test_one : failed - fail!"])
```

Comment:
break
}
// Otherwise, wait another fraction of a second.
runLoop.runUntilDate(NSDate(timeIntervalSinceNow: 0.01))
} while NSDate().compare(timeoutDate) == NSComparisonResult.OrderedAscending
if unfulfilledDescriptions.count > 0 {
// Not all expectations were fulfilled. Append a failure
// to the array of expectation-based failures.
let descriptions = unfulfilledDescriptions.joined(separator: ", ")
- let failure = XCTFailure(
- message: "Exceeded timeout of \(timeout) seconds, with unfulfilled
expectations: \(descriptions)",
- failureDescription: "Asynchronous wait failed",
- expected: true,
- file: file,
- line: line)
- if let failureHandler = XCTFailureHandler {
- failureHandler(failure)

2016-04-03 21:32 GMT+02:00 Brian Croom <notifications@github.com>:

> In Tests/Functional/Observation/All/main.swift
> <https://github.com/apple/swift-corelibs-xctest/pull/86#discussion_r58316115>
> :
>
> > @@ -74,10 +78,10 @@ class Observation: XCTestCase {
> >          XCTAssertEqual(observer.finishedBundlePaths.count, 0)
> >
> >          XCTFail("fail!")
> > -        XCTAssertEqual(observer.failureDescriptions, ["failed - fail!"])
> > +        XCTAssertEqual(observer.failureDescriptions, ["Observation.test_one : failed - fail!"])
>
> This is the test change I was referring to in the comment above
> <https://github.com/apple/swift-corelibs-xctest/pull/86/files#r58315910>.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/apple/swift-corelibs-xctest/pull/86/files/651fd407f55f25eba400bf19003553f236e22083#r58316115>
>